keps/sig-auth/0000-20170814-bounding-self-labeling-kubelets.md

```diff
@@ -105,8 +105,35 @@ and warns when invoked with `kubernetes.io` or `k8s.io` labels outside that set.
 * NodeRestriction admission prevents kubelets from adding/removing/modifying `kubernetes.io` or `k8s.io`
 labels other than the specifically allowed labels/prefixes described above on Node *update* only
 
-v1.15:
-
+v1.14:
+
+* Begin migration/removal of in-tree `--node-labels` use outside of the allowed set by addons:
+  * `beta.kubernetes.io/fluentd-ds-ready`
+    * addon: remove from the nodeSelector
+    * kube-up: remove from the default `--node-labels` flag
+  * `beta.kubernetes.io/metadata-proxy-ready`
+    * addon: announce the nodeSelector will switch to `cloud.google.com/metadata-proxy-ready` in 1.15
```

Comment:
Or do something in the gcp-controller-manager node annotator. I'd like to deprecate this anyway. `cloud.google.com/metadata-proxy-ready` should probably be something closer to  `security.gke.io/metadata-proxy-ready` if we go this route.